h/views/api_auth.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from h.auth.services import TOKEN_TTL
+from h.exceptions import OAuthTokenError
+from h.util.view import json_view
+
+
+@json_view(route_name='token', request_method='POST')
```

Comment:
Alright, I just found the documentation again. Access Token requests according to [RFC6749](https://tools.ietf.org/html/rfc6749#section-4.1.3) need to be `application/x-www-form-urlencoded`. So I won't make this endpoint work with JSON.